Exercise_A/index.html

```diff
@@ -15,7 +15,12 @@
     />
   </head>
   <body>
-    <!-- CODE HERE -->
+    
+    <a href = "about.html">
+      <img src = "drink3.jpg">
+    </a>
+    
+    
```

Comment:
Nice work Ex A acts as intended! Consider removing boiler plate code you will not use